FreeRTOS-Plus/Demo/MQTT_Mutual_Auth_Qemu_mps2/scripts/mps2_m3.ld

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
Does this file differ from the other qemu demos?

Reply:
I am not sure about other demos, but it does not differe from the freertos + tcp qemu mps2 demo